how-to.md

```diff
@@ -0,0 +1,194 @@
+# How to WP-CLI
+
+## Introduction
+
+Using WP-CLI you can perform specific tasks using the command line instead of accessing the WordPress admin panel. In the following section we will cover step by step some of these
+tasks.
+
+## How to install WordPress
+
+Download and install WordPress using WP-CLI is very easy. It's done in four simple steps.
+First you will need to download WordPress using the `wp core download` command.
+
+### Step 1 - Download WordPress
+
+The syntax of the command to download WordPress is the following `wp core download [--path=<path>] [--locale=<locale>] [--version=<version>] [--skip-content] [--force]`
+
+    $ wp core download --path=wpdemo.test --locale=it_IT
+    Creating directory '/wpdemo.test/'.
+    Downloading WordPress 5.4.1 (it_IT)...
+    md5 hash verified: 3fa03967b47cdfbf263462d451cdcdb8
+    Success: WordPress downloaded.
+
+The command above creates a wpdemo.test/ folder inside your current working directory and downloads the latest WordPress version. You can replace the --path=wpdemo.test with your
```

Comment:
Everything that is console input/output or filenames/commands should be written as code blocks (wrapped in one backtick).